web/server/vue-cli/src/assets/userguide/userguide.md

```diff
@@ -2,9 +2,9 @@ User Guide
 ==========
```

Comment:
Should we highlight that this is the user guide for the web UI?

Maybe links from here to the CLI user guide or from the CLI user guides to here would be good.